pkg/apps/git.go

```diff
@@ -16,103 +16,116 @@ import (
 	git "gopkg.in/src-d/go-git.v4"
 	gitObj "gopkg.in/src-d/go-git.v4/plumbing/object"
 	gitSt "gopkg.in/src-d/go-git.v4/storage/filesystem"
-	gitFS "gopkg.in/src-d/go-git.v4/utils/fs"
+	gitFS "srcd.works/go-billy.v1"
```

Comment:
@nono I don't know why they did not use `gopkg.in`. I can use the [github address](https://github.com/src-d/go-billy) instead, if you prefer...